subjects/iscapitalized/README.md

```diff
@@ -0,0 +1,49 @@
+## iscapitalized
+
+### Instructions
+
+Write a function `IsCapitalized()` that takes a `string` as an argument and returns `true` if each word in the `string` begins with either an uppercase letter or a non-alphabetic character.
+
+- If any of the words begin with a lowercase letter return `false`.
+- If the `string` is empty return `false`.
+
```

Comment:
Should we had a case for when string is `"        "` ?

Reply:
It is already explicit in the instruction.